flows/actions/modifiers/urn.go

```diff
@@ -27,7 +27,8 @@ const (
 	URNRemove URNModification = "remove"
 )
 
-// URNModifier modifies a URN on a contact
+// URNModifier modifies a URN on a contact. This has been replaced by URNsModifier but is kept here for now
```

Comment:
we could eventually drop this once Surveyor has been updated enough time has passed that we're not worried about old submissions out there using this modifier type

Reply:
But wouldn't that get us in trouble? I thought the idea of modifiers is just that, to modify, not set, otherwise a surveyor submission might override URNs that have been set from the website instead of only adding a new URN.

Reply:
So this new modifier can `append` / `remove` / `set` 
- flow actions are still going to use `append`
- requests from the RP UI are going to use `set`

Reply:
Oh I see, ok great.